cockpit_api/src/main/java/com/cockpit/api/service/jiragateway/JiraGatewayService.java

```diff
@@ -32,27 +37,38 @@
 import java.io.IOException;
 import java.util.*;
 
-import static javax.management.timer.Timer.*;
+import static javax.management.timer.Timer.ONE_SECOND;
+import static javax.management.timer.Timer.ONE_MINUTE;
 
 @Configuration
 @EnableScheduling
 @Service
 public class JiraGatewayService {
-	
-    private ModelMapper modelMapper = new ModelMapper();
+
+    private final ModelMapper modelMapper = new ModelMapper();
+
     private final SprintRepository sprintRepository;
     private final JiraRepository jiraRepository;
     private final UserStoryRepository userStoryRepository;
+    private final UserStoryService userStoryService;
```

Comment:
Unused "private" fields should be removed: private final UserStoryService userStoryService;